api/v2/openapi.yaml

```diff
@@ -0,0 +1,345 @@
+---
+
+swagger: '2.0'
+
+info:
+  version: 0.0.1
+  title: Alertmanager API
+  description: API of the Prometheus Alertmanager (https://github.com/prometheus/alertmanager)
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+
+paths:
+  /status:
+    get:
+      tags:
+        - general
+      operationId: getStatus
+      description: Get current status of an Alertmanager instance and its cluster
+      responses:
+        '200':
+          description: Status response
+          schema:
+            $ref: '#/definitions/alertmanagerStatus'
+  /receivers:
+    get:
+      tags:
+        - receiver
+      operationId: getReceivers
+      description: Get list of all receivers (name of notification integrations)
+      responses:
+        '200':
+          description: Receivers response
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/receiver'
+  /silences:
+    get:
+      tags:
+        - silence
+      operationId: getSilences
+      description: Get a list of silences
+      responses:
+        '200':
+          description: List of silences response
+          schema:
+            $ref: '#/definitions/silences'
+        '500':
+          description: Internal server error
+          schema:
+            type: string
+      parameters:
+        - name: filter
+          in: query
+          description: A list of matchers to filter silences by
+          required: false
+          type: array
+          items:
+            type: string
+    post:
+      tags:
+        - silence
+      operationId: postSilences
+      description: Post a new silence or update an existing one
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          name: silence
+          description: the silence to create
+          required: true
+          schema:
+            $ref: '#/definitions/silence'
+      responses:
+        '200':
+          description: Created / updated silence response
+          schema:
+            type: object
+            properties:
+              silenceID:
+                type: string
+        '400':
+          description: Bad request
+          schema:
+            type: string
+  /silence/{silenceID}:
+    get:
+      tags:
+        - silence
+      operationId: getSilence
+      description: Get a silence by its id
+      parameters:
```

Comment:
Since `silenceID` is a `path` parameter, you could move this `parameters` block under `/silence/{silenceID}` so that it applies to all methods.

Reply:
:+1: